app/views/external_activities/_form.html.haml

```diff
@@ -52,6 +52,11 @@
     = field_set_tag 'Student Report Enabled' do
       = f.check_box :student_report_enabled
       = t('authoring.student_report_enabled_description')
+
+    = field_set_tag 'Archive' do
+      = f.check_box :is_archived
+      This material is archived
+
```

Comment:
I think the archiving of a material should be a button. It is kind of like 'deleting' because it hides the material from most places. So it makes more sense that is a button instead of a checkbox that then requires saving.  There is a button on the 'matedit?iframe=true' path, so I think that same button could be added here.